controllers/v1/observationSubmissionsController.js

```diff
@@ -216,6 +218,28 @@ module.exports = class ObservationSubmissions extends Abstract {
           isRubricDriven: solutionDocument.isRubricDriven
       };
 
+      //Fetch user profile information by calling sunbird's user read api.
+      let userProfileData = {};
+      let userProfile = await sunbirdUserProfile.profile(req.userDetails.userToken, req.userDetails.userId);
```

Comment:
@VISHNUDAS-tunerlabs  this is not required observation will always have profile we should capture from that in submission.